plugin/src/main/cpp/classes/openxr_fb_hand_tracking_mesh.cpp

```diff
@@ -39,19 +39,26 @@
 
 using namespace godot;
 
-void OpenXRFbHandTrackingMesh::setup_hand_mesh(Hand p_hand) {
-	if (p_hand != hand) {
+void OpenXRFbHandTrackingMesh::setup_hand_mesh(bool p_success, const Ref<Mesh> &p_mesh) {
+	if (!p_success) {
+		emit_signal("openxr_fb_hand_tracking_mesh_unavailable");
 		return;
 	}
```

Comment:
Seems like we can remove the `p_success` argument either by checking if `p_mesh.is_valid()` or by emitting the signal when hand mesh fetch fails outside of this method.